tutorials/plot_background_filtering.py

```diff
@@ -894,6 +894,104 @@ def baseline_plot(x):
+# :func:`mne.filter.create_filter` to print out the settings once more:
+
+# Use the same settings as when calling e.g., `raw.filter()`
+fir_coefs = mne.filter.create_filter(data=None,  # Data is only used for sanity checking, not strictly needed  # noqa
+                                     sfreq=1000.,  # sfreq of your data in Hz
+                                     l_freq=None,
+                                     h_freq=40.,  # assuming a lowpass of 40 Hz
+                                     method='fir',
+                                     fir_window='hamming',
+                                     fir_design='firwin',
+                                     verbose=True)
+
+# See the printed log for the transition bandwidth and filter length
+# Alternatively, get the filter length through:
+filter_length = fir_coefs.shape[0]
+
+###############################################################################
+# .. note:: If you are using an IIR filter, :func:`mne.filter.create_filter`
+#           will not print a filter length and transition bandwidth to the log.
+#           Instead, you can specify the roll off with the `iir_params`
+#           argument or stay with the default, which is a 4th order
+#           (Butterworth) filter.
+#
+# Passband ripple and stopband attenuation
+# ----------------------------------------
+#
+# When use standard :func:`scipy.signal.firwin` design (as for FIR filters in
+# MNE), the passband ripple and stopband attenuation are dependent upon the
+# window used in design. For standard windows the values are listed in this
+# table (see Ifeachor & Jervis, p. 357 [3]_):
+#
+# +-------------------------+-----------------+----------------------+
+# | Name of window function | Passband ripple | Stopband attenuation |
+# +=========================+=================+======================+
+# | Hann                    | 0.0545 dB       | 44 dB                |
+# +-------------------------+-----------------+----------------------+
+# | Hamming                 | 0.0194 dB       | 53 dB                |
+# +-------------------------+-----------------+----------------------+
+# | Blackman                | 0.0017 dB       | 74 dB                |
+# +-------------------------+-----------------+----------------------+
+#
+#
+# Filter delay and direction of computation
+# -----------------------------------------
+# For reporting this information, it might be sufficient to read the docstring
+# of the filter function or method that you apply. For example in the
+# docstring of `mne.filter.create_filter`, for the phase parameter it says:
+#
+#    Phase of the filter, only used if ``method='fir'``.
+#    By default, a symmetric linear-phase FIR filter is constructed.
+#    If ``phase='zero'`` (default), the delay of this filter
+#    is compensated for. If ``phase=='zero-double'``, then this filter
+#    is applied twice, once forward, and once backward. If 'minimum',
```

Comment:
Super nitpick: these lines are too short, right?

(The term is "hemineglect")

Reply:
Haha yes, I copied that from a docstring and did not adjust the linewidths

Reply:
That's fine, will make any future copy-paste updates easier